codecov.yml

```diff
@@ -0,0 +1,3 @@
+ignore:
+  - "src/umpire/tpl"
```

Comment:
Won't this also ignore simPool?  I think we want to add that right?

Reply:
Yes. I can change that.